easybuild/easyconfigs/g/GDRCopy/GDRCopy-2.1-GCCcore-9.3.0-CUDA-11.0.2.eb

```diff
@@ -0,0 +1,50 @@
+easyblock = 'ConfigureMake'
+
+name = 'GDRCopy'
+version = '2.1'
+local_cudaversion = '11.0.2'
+versionsuffix = '-CUDA-%s' % local_cudaversion
+
+homepage = 'https://github.com/NVIDIA/gdrcopy'
+description = "A low-latency GPU memory copy library based on NVIDIA GPUDirect RDMA technology."
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+toolchainopts = {'pic': True}
+
+github_account = 'NVIDIA'
+source_urls = [GITHUB_SOURCE]
+sources = ['%(version)s.tar.gz']
+checksums = ['cecc7dcc071107f77396f5553c9109790b6d2298ae29eb2dbbdd52b2a213e4ea']
+
+builddependencies = [
+    ('binutils', '2.34'),
+    ('Autotools', '20180311'),
+    ('pkg-config', '0.29.2'),
+]
+
+osdependencies = [
+    # Kernel module for GDR Copy: improves Host to GPU communication
+    ('gdrcopy-kmod', 'gdrdrv-dkms'),
+    # Kernel module for GPUDirect RDMA: improves GPU to GPU communication
+    # Requires a Mellanox HCA with MLNX_OFED 2.1
+    # https://github.com/Mellanox/nv_peer_memory
+    'nvidia_peer_memory',
+]
+
+dependencies = [
+    ('CUDAcore', local_cudaversion, '', True),
+    ('Check', '0.15.2'),
+]
+
+skipsteps = ['configure']
+
+local_envopts = "PREFIX=%(installdir)s CUDA=$EBROOTCUDACORE"
+buildopts = "all %s" % local_envopts
```

Comment:
If I replace this by
`buildopts = "config lib exes %s" % local_envopts`
it compiles everything but the kernel driver (which I don't think belongs in a module anyway) and compiles on a regular non-GPU host.

Reply:
@bartoldeman that's indeed a good point, the kernel modules is not needed and disabling its build simplifies a lot the requirements of this easyconfig